TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
+    {
+
+        private static readonly string pg_connectionString = "Host=localhost;Username=admin;Password=12345678;Database=easyrest";
+
+        public static void ShowHeaders(NpgsqlDataReader reader)
+        {
+            var columnNames = Enumerable.Range(0, reader.FieldCount)
+                .Select(reader.GetName)
+                .ToArray();
+            foreach (var column in columnNames) 
+            { 
+                Console.Write($"{column}\t"); 
+            }
+            Console.WriteLine();
+        }
+
+        public static void ShowRows(NpgsqlDataReader reader)
```

Comment:
should it be public if this class is a wrapper?

Reply:
Must be private.